torch_tvm/compiler.cpp

```diff
@@ -148,7 +152,15 @@ tvm::relay::Function TVMCompiler::convertToRelay(
       auto uses = value->uses();
       for (const auto& use : uses) {
         tvm::Array<tvm::relay::Expr> relay_inputs;
+        // Things like prim::Return
+        // Should we be more explicit here?
+        // That only prim::Return should be skipped?
+        if (use.user->outputs().size() < 1) {
```

Comment:
what is this logic doing here?

Reply:
I just moved the logic up here. It was always there. We do not translate return to any tvm op. So we dont translated it.